app/assets/javascripts/discourse/app/templates/preferences/categories.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="control-group category-notifications">
   <label class="control-label">{{i18n "user.categories_settings"}}</label>
 
-  <div class="controls tracking-controls">
+  <div class="controls tracking-controls tracking-controls--watched-categories">
```

Comment:
`--` is this related to any pattern we are following?

Reply:
It represents the modifier syntax when following BEM

Reply:
I thought there was a space in this case (different class) but apparently not, ok then 🤷‍♂️ Swear